acceptance/tests/pxp-module-puppet/run_puppet_exec.rb

```diff
@@ -0,0 +1,87 @@
+require 'pxp-agent/test_helper.rb'
+require 'puppet/acceptance/environment_utils'
+
+test_name 'C98107 - Run puppet with an exec' do
+
+  extend Puppet::Acceptance::EnvironmentUtils
+
+  env_name = test_file_name = File.basename(__FILE__, '.*')
+  environment_name = mk_tmp_environment(env_name)
+
+  step 'On master, create a new environment that execs hostname' do
+    site_manifest = "#{environmentpath}/#{environment_name}/manifests/site.pp"
+    create_remote_file(master, site_manifest, <<-SITEPP)
+node default {
+  exec { "hostname":
+    path => ["/bin", "/usr/bin", "C:/cygwin32/bin", "C:/cygwin64/bin"],
```

Comment:
@MikaelSmith this is failing CI when running 32-bit puppet/ruby on 32-bit windows (using the `win-10-ent-i386` template), because the path needs to be `C:/cygwin/bin`. I think the `C:/cygwin32` would only happen when using 32-bit cygwin on 64-bit windows, which shouldn't happen in our current set of windows templates.